DataCollector/RulerDataCollector.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+
+use Hoathis\Bundle\RulerBundle\Ruler\LoggedRuler;
```

Comment:
Hum, I used the [BenchBundle](https://github.com/hoaproject/Contributions-Symfony-BenchBundle) as an inpiration and it does not follow this rule. The console bridge does though.

I can use this pattern if you want but I find it a bit weird. Having both *Symfony* and *Bundle* in the namespace does not feel right as a bundle is almost always related to Symfony. WDYT?